### PR TITLE
PRIME-2061 Licence refactor with details

### PR DESCRIPTION
**Pull request by hogmann**
To test:

- verify lookup returns the same data 
- work through the enrollee workflow and add a college certificate
- verify there are no errors in the rules engine and the proper automatic or manual status is achieved.

### CODE REVIEW

**Review by AMiKaH — COMMENTED** [review]
LGTM

**Review by AMiKaH — APPROVED** [review]

**Review by Paahn — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/Models/Certifications/LicenseDetail.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    [Table("LicenseDetail")]
+    public class LicenseDetail : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public int LicenseCode { get; set; }
+
+        [NotMapped]
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/Certifications/LicenseDetail.cs:17]: why is this marked `[NotMapped]`?

File: prime-dotnet-webapi/Models/Certifications/LicenseDetail.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+namespace Prime.Models
+{
+    [Table("LicenseDetail")]
+    public class LicenseDetail : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/Certifications/LicenseDetail.cs:14]: `int` isn't nullable, so you don't need `[Required]`.

**Pull request merged**